clang/test/Frontend/raw_string_line_number.cpp

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang_cc1 -E %s | FileCheck %s
+// CHECK: AAA
+// CHECK-NEXT: BBB
+R"(
+AAA)"
+BBB
```

Comment:
This test is not testing line numbers

Reply:
This test demonstrates that the line `AAA` is followed immediately by the line `BBB` with no intervening blank line. Without the fix, there is an incorrect blank line between them. This test fails without the fix.

Reply:
Thanks for the explanation! Perhaps a better name for the test file is `raw_string_print.cpp` or something along those lines, so nobody is wondering what this has to do with line numbers?